pre-docs/plan_functions.md

```diff
@@ -481,36 +527,21 @@ run_task('facts', $targets, '_run_as' => 'root')
 ### Run a task, logging the provided description.
 
 ```
-run_task(String[1] $task_name, Boltlib::TargetSpec $targets, String $description, Optional[Hash[String[1], Any]] $task_args)
+run_task(String[1] $task_name, Boltlib::TargetSpec $targets, Optional[String] $description, Optional[Hash[String[1], Any]] $task_args)
 ```
 
 *Returns:* `ResultSet` A list of results, one entry per target.
 
 * **task_name** `String[1]` The task to run.
 * **targets** `Boltlib::TargetSpec` A pattern identifying zero or more targets. See [`get_targets`](#get_targets) for accepted patterns.
-* **description** `String` A description to be output when calling this function.
+* **description** `Optional[String]` A description to be output when calling this function.
 * **task_args** `Optional[Hash[String[1], Any]]` Arguments to the plan. Can also include additional options: '_catch_errors', '_run_as'.
 
 **Example:** Run a task
 ```
 run_task('facts', $targets, 'Gather OS facts')
 ```
 
-### Run a task, calling the block as each node starts and finishes execution. This is used from 'bolt task run'
```

Comment:
I guess we forgot to run the docs rake task for https://github.com/puppetlabs/bolt/pull/955/commits/377350ecf2a9e193b1013fb05554cd12e7418245